plugins/kosync.koplugin/main.lua

```diff
@@ -919,6 +921,17 @@ function KOSync:onKOSyncPullProgress()
     self:getProgress(true, true)
 end
 
+function KOSync:onKOSyncToggleAutoSync()
+    self.settings.auto_sync = not self.settings.auto_sync
+    local notify_text
+    if self.settings.auto_sync then
+        notify_text = _("Auto Sync ON")
+    else
+        notify_text = _("Auto Sync OFF")
```

Comment:
```suggestion
        notify_text = _("Auto-progress sync on")
    else
        notify_text = _("Auto-progress sync off")
```
I'm mainly referring to the capitalization.

Reply:
I would add a colon. "Auto progress sync: on/off". Otherwise it looks a bit like an unfinished sentence. (Same style I used in a39d612f1831379ea5fc2675b7dcd10a67de5ccd, where I faced the same dilemma.)